From 787f872d810fab93d806b26916efd6ca82283122 Mon Sep 17 00:00:00 2001
From: Kyle Fuller <kyle@fuller.li>
Date: Wed, 9 Dec 2015 17:33:08 +0000
Subject: [PATCH] Conform to PEP 0394 in Python sources

> In order to tolerate differences across platforms, all new code that
> needs to invoke the Python interpreter should not specify python ,
> but rather should specify either python2 or python3
---
 Utilities/bootstrap | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Utilities/bootstrap b/Utilities/bootstrap
index 11faaf4..e8abbea 100755
--- a/Utilities/bootstrap
+++ b/Utilities/bootstrap
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 """
  This source file is part of the Swift.org open source project
-- 
2.6.3

